pkg/apis/aadpodidentity/v1/types.go

```diff
@@ -31,6 +31,9 @@ const (
 
 // AzureIdentity is the specification of the identity data structure.
 //+k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+//+kubebuilder:printcolumn:name="Type",type="string",JSONPath=".spec.type",description="",priority=1
```

Comment:
I am thinking maybe we can assign the priority to 0 so we don't have to append `-owide` during `kubectl get` to get the additional information. What do you think?

Reply:
Yep, makes sense to me as they should be present on all resources of this type 👍 I've updated the PR to reflect this.